### ISSUE TITLE
Has clashing colors with Gitsigns

### ISSUE THREAD

**Issue by AlphabetsAlphabets** [habamax/vim-habamax#2]
![image](https://user-images.githubusercontent.com/44822875/217667940-24a16756-a801-4503-80bd-0b883c0baa2f.png)

Hi I've been really enjoying your theme. I only have one problem with it and it is with the plugin called gitsigns. As you can see the colors go wacky. Are you willing to add in changes to accommodate gitsigns?

**Comment by habamax** [comment]
I don't use gitsigns so not 100% sure what is going on here, but you can do usual thing (overriding some specific highlights) as described here: https://github.com/vim/colorschemes/wiki/How-to-override-a-colorscheme%3F

**Comment by AlphabetsAlphabets** [comment]
> I don't use gitsigns so not 100% sure what is going on here, but you can do usual thing (overriding some specific highlights) as described here: https://github.com/vim/colorschemes/wiki/How-to-override-a-colorscheme%3F

I've tried overriding it but I only have minimal success. I've been trying for a few hours already.

**Comment by AlphabetsAlphabets** [comment]
I found this weird behavior where if I apply a color scheme before habamax, that colorscheme's highlight groups will apply. This probably just a band aid solution until some patch comes out and this doesn't work anymore.

**Comment by habamax** [comment]
> I've tried overriding it but I only have minimal success. I've been trying for a few hours already.

It should be easy to do once you have identified all the groups to override unless plugin author made it more complex than needed.

I don't speak lua so have no idea what is happening there:
![image](https://user-images.githubusercontent.com/234774/217728695-0223ae2e-3274-44a3-ac73-b2709719a342.png)

But you should be able to get those group names and override them.

if this would not interfere... :)

![image](https://user-images.githubusercontent.com/234774/217728926-967bb9e7-7dba-4573-bce2-a96391e52dac.png)

**Comment by AlphabetsAlphabets** [comment]
> I don't speak lua so have no idea what is happening there:

Neither do I, I managed to find the highlight groups I want to change by just looking at the output of `:verbose hi` and finding the colors sample text that is the most difficult to read. I've changed those options but, I am not having any luck whatsoever.

I changed the colors of the highlight group with `:hi <group> ctermfg=...` but I have no idea what's going on. I'll ask around with the Gitsigns community. Thanks for your help, as I realized I should've gone there to begin with.
